ai/cohere/v0/README.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Cohere"
+lang: "en-US"
+draft: false
+description: "Learn about how to set up a VDP Cohere component https://github.com/instill-ai/instill-core"
+---
+
+The Cohere component is an AI component that allows users to connect the AI models served on the Cohere Platform.
+It can carry out the following tasks:
+
+- [Text Generation Chat](#text-generation-chat)
+- [Text Embeddings](#text-embeddings)
+- [Text Reranking](#text-reranking)
+
+
+
+## Release Stage
+
+`Alpha`
+
+
+
+## Configuration
+
+The component configuration is defined and maintained [here](https://github.com/instill-ai/component/blob/main/ai/cohere/v0/config/definition.json).
+
+
+
+
+## Setup
+
+
+| Field | Field ID | Type | Note |
+| :--- | :--- | :--- | :--- |
+| API Key (required) | `api-key` | string | Fill your Cohere API key. To find your keys, visit the Cohere dashboard page. |
+
+
+
+
+## Supported Tasks
+
+### Text Generation Chat
+
+Provide text outputs in response to text inputs.
+
+
+| Input | ID | Type | Description |
+| :--- | :--- | :--- | :--- |
+| Task ID (required) | `task` | string | `TASK_TEXT_GENERATION_CHAT` |
+| Model Name (required) | `model-name` | string | The Cohere command model to be used. |
+| Prompt (required) | `prompt` | string | The prompt text |
+| System message | `system-message` | string | The system message helps set the behavior of the assistant. For example, you can modify the personality of the assistant or provide specific instructions about how it should behave throughout the conversation. By default, the model’s behavior is using a generic message as "You are a helpful assistant." |
+| Documents | `documents` | array[string] | The documents to be used for the model, for optimal performance, the length of each document should be less than 300 words. |
+| Extra Parameters | `extra-params` | object | Extra Parameters |
```

Comment:
```suggestion
| Extra Parameters | `extra-params` | object | Extra parameters |
```

Reply:
@donch1989 We can remove `Extra parameters` right?

Reply:
Fixed